internal/controllers/machinedeployment/mdutil/util_test.go

```diff
@@ -317,11 +324,18 @@ func TestFindNewMachineSet(t *testing.T) {
 	oldMS := generateMS(deployment)
 	oldMS.Spec.Template.Spec.InfrastructureRef.Name = "changed-infra-ref"
 
+	msCreatedBeforeRolloutAfter := generateMS(deployment)
+	msCreatedBeforeRolloutAfter.CreationTimestamp = twoBeforeRolloutAfter
```

Comment:
nit: please rename to msCreatedTwoBeforeRolloutAfter (same for the next one)
Just makes it easier to read the test cases

(same for the next unit test)